### PR TITLE
cocoapods 0.36.0 pre-release instructions

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

**Comment by corinnekrych** [comment]
@cvasilak 
do you mind to review?

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,13 @@ xcode_project: AGURLSessionStubs.xcodeproj
 xcode_scheme: AGURLSessionStubs
 xcode_sdk: iphonesimulator
 
+before_install:
+    - gem install cocoapods --pre --no-rdoc --no-ri --no-document --quiet
+
+script:
+    - set -o pipefail && xcodebuild -workspace AGURLSessionStubs.xcworkspace -scheme AGURLSessionStubs -sdk iphonesimulator -configuration Release clean test
```
Comment (cvasilak) [inline .travis.yml]: if ```xcodebuild``` is required to I am fine including it, if not  let's remove it and use the provided ```xctool```

**Comment by cvasilak** [comment]
I think since there are 0 dependencies, let's not include a Podfile but +1 for removing the Gemfile et like

File: README.md [hunk 2]
```diff
@@ -55,19 +55,19 @@ StubsManager.stubRequestsPassingTest({ (request: NSURLRequest!) -> Bool in
...
 
 and then:
+
 ```bash
-bundle install
-bundle exec pod install
+pod install
 ```
 
 to install your dependencies
```
Comment (cvasilak) [inline README.md]: let remove the 'Drag the library in your project'  and reuse [the instructions](https://github.com/corinnekrych/aerogear-ios-httpstub/blob/cocopods.0.36.support/README.md) we have already in the http-lib for the "Manual Installation"

Comment (cvasilak) [inline README.md]: when doing it let's remove the 'AGURLSessionStubs_usage.png' image we have in the repo

File: .travis.yml [hunk 3]
```diff
@@ -4,6 +4,13 @@ xcode_project: AGURLSessionStubs.xcodeproj
 xcode_scheme: AGURLSessionStubs
 xcode_sdk: iphonesimulator
 
+before_install:
+    - gem install cocoapods --pre --no-rdoc --no-ri --no-document --quiet
```
Comment (cvasilak) [inline .travis.yml]: if the ```Podfile``` is removed let's remove the ```before_install:``` step

**Comment by corinnekrych** [comment]
@cvasilak +1 I think having consistent README across repos make sense. 
With the same consistency effort, using workspace for all projects is better, same for using consistent travis script

**Comment by cvasilak** [comment]
:+1: after the last updates

**Comment by cvasilak** [comment]
thank you @corinnekrych :rocket: landed on https://github.com/aerogear/aerogear-ios-httpstub/commit/8376231022843716a4e4ce319eb8d49f6317cbba

**Pull request merged**